tests/snappi_tests/multidut/pfcwd/test_multidut_pfcwd_basic_with_snappi.py

```diff
@@ -172,8 +172,10 @@ def test_pfcwd_basic_multi_lossless_prio_reboot(snappi_api,                 # no
                                                 localhost,
                                                 duthosts,
                                                 enum_dut_lossless_prio_with_completeness_level,   # noqa: F811
```

Comment:
@rraghav-cisco   'duthosts' and 'enum_dut_lossless_prio_with_completeness_level' are not used for this test case anymore. pls remove these two.

Reply:
Done.